src/snowflake/cli/api/project/schemas/native_app/native_app.py

```diff
@@ -24,6 +24,10 @@ class NativeApp(UpdatableModel):
         title="Folder at the root of your project where the build step copies the artifacts.",
         default="output/deploy/",
     )
+    generated_root: Optional[str] = Field(
+        title="Subdirectory of the deploy root where files generated by the Snowflake CLI will be written.",
+        default="__generated/",
```

Comment:
Any reason we don't want this nested under the `output/` directory? To me, this is output.

Reply:
it's under `output/deploy/__generated` now because it needs to end up on the stage.

Reply:
Oh. I swear "Subdirectory of deploy root" wasn't there when I reviewed 😂 ignore this

Reply:
All files under this generated directory also get uploaded to the stage, with the `__generated` parent directory preserved and visible on the stage. Which means if `output/deploy` mirrors the stage, then `__generated` should belong directly under `deploy`.